src/components/modals/add-to-collection.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { updateCollection } from '../../internal/collection';
+import { setWishlist } from '../../internal/wishlist';
+import Modal from '../modal';
+
+const AddToCollectionModal = (props) => {
+  const [selectedCollection, setSelectedCollection] = useState(props.collections[0].id);
+
+  const addCapToCollection = () => {
+    const collection = props.collections.find((c) => c.id === selectedCollection);
+    const { id, name } = props.cap;
+
+    const has = collection.content.items.find((i) => i.id === id);
+    if (!has) {
+      collection.content.items.push({ id, legend: name, prio: false });
+      if (props.authorized) {
+        updateCollection(selectedCollection, {
+          name: collection.name,
+          wishlist: collection.content,
+        })
+          .then(() => {
+            props.setSuccessAlert(true);
```

Comment:
We might want to know if it has been synced to cloud or not.

Reply:
if the user already logged in, it will be synced to cloud. else, it save to local storage as normal